### ISSUE TITLE
Can the position of the ShellItems in the TabBar be adjusted?

### ISSUE THREAD

**Issue by TheBaileyBrew** [VladislavAntonyuk/vladislavantonyuk.github.io#77]
Hello!

I've been working through understanding all the work you did here: https://vladislavantonyuk.github.io/articles/Adding-custom-action-button-to-.NET-MAUI-Shell-TabBar/

And I love it!! It's exactly the concept that I was looking for instead of going the route of doing something like Sharpnado, Telerik or Syncfusion.

I have successfully implemented the handlers/renderers in my app, but I'm wondering if there's a way to adjust the spacing of the tabs themselves, since I have a need for 4 tabs to be visible, and it makes the middle 2 tabs REALLY close to the center action button?

![image](https://github.com/VladislavAntonyuk/vladislavantonyuk.github.io/assets/35344621/5f4e6105-a7c5-40ce-8f09-52ddcedda999)

**Comment by TheBaileyBrew** [comment]
I made it work by just adding an empty tab, but still curious if there's a way to do it via one of the renderers..

**Comment by VladislavAntonyuk** [comment]
yes, you have full control how to position the items. You only need to change the space between items.
as for middle view, I set position Center:             middleView.Center = new CGPoint(View.Bounds.GetMidX(), TabBar.Frame.Top - middleView.Frame.Height / 2);
You can adjust it according your needs

**Comment by rpharon** [comment]
@TheBaileyBrew Hi, have you figured out how to space out each items based on @VladislavAntonyuk comments?
